src/transformers/modeling_tf_utils.py

```diff
@@ -808,20 +839,29 @@ def prune_heads(self, heads_to_prune):
         """
         raise NotImplementedError
 
-    def save_pretrained(self, save_directory):
+    def save_pretrained(self, save_directory, saved_model=True, version=1):
         """
         Save a model and its configuration file to a directory, so that it can be re-loaded using the
         :func:`~transformers.TFPreTrainedModel.from_pretrained` class method.
 
         Arguments:
             save_directory (:obj:`str`):
                 Directory to which to save. Will be created if it doesn't exist.
+            saved_model (:obj:`bool`, `optional`, defaults to False):
+                If the model has to be saved in saved model format as well or not.
+            version: (:obj:`int`, `optional`, defaults to 1):
+                The version of the saved model.
```

Comment:
What is a version of a model? This looks dangerous

Reply:
Do we need that?

Reply:
One of the advantages of the saved models is that they can be versioned. The version is very important as it is required to be loaded by TF serving, otherwise the server raise an error.

Reply:
If it's a necessary parameter required by TF serving then ok for me.

Reply:
Could we expand the description and link to the relevant doc page on the TF serving side?